docs/basic-docs/atomicdex/atomicdex-desktop/add-coin-to-atomicdex-desktop.md

```diff
@@ -0,0 +1,259 @@
+# Adding a Coin on AtomicDEX Desktop
+
+Before a coin can be listed in AtomicDEX, it needs to be confirmed for compatibility, and present within the [Komodo Platform coins repository](https://github.com/KomodoPlatform/coins).
+
+This involves:
+- Setting up two operational [electrum servers](https://electrumx.readthedocs.io/en/latest/)
```

Comment:
Please change the link to https://electrumx-spesmilo.readthedocs.io/en/latest/
this one points to the old repo, the new one is https://github.com/spesmilo/electrumx

Reply:
done